azurecontainerapps/azurecontainerapps.ts

```diff
@@ -221,12 +217,13 @@ export class azurecontainerapps {
             telemetryHelper.setSuccessfulResult();
         } catch (err) {
             tl.setResult(tl.TaskResult.Failed, err.message);
+            telemetryHelper.setFailedResult();
```

Comment:
It would be great if we also logged the error message, so that if we see errors happening we can know how to address them. Maybe we could log result = "failed: <ERROR>"? Or create a new error log in whatever way we usually do it in our telemetry?

Reply:
That's a good idea -- let me add that as a separate property as a part of the event.

I'll need to figure out a way to grab error messages in the GitHub Action, but that can be a later change.